addons/mrp_subcontracting/security/ir.model.access.csv

```diff
@@ -0,0 +1,23 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_subcontracting_portal_stock_picking,subcontracting.portal.picking,stock.model_stock_picking,base.group_portal,1,0,0,0
+access_subcontracting_portal_picking_type,subcontracting.portal.picking.type,stock.model_stock_picking_type,base.group_portal,1,0,0,0
+access_subcontracting_portal_stock_move,subcontracting.portal.move,stock.model_stock_move,base.group_portal,1,1,1,0
+access_subcontracting_portal_stock_move_line,subcontracting.portal.move.line,stock.model_stock_move_line,base.group_portal,1,1,1,1
+access_subcontracting_portal_scrap,subcontracting.portal.scrap,stock.model_stock_scrap,base.group_portal,1,0,0,0
+access_subcontracting_portal_warehouse,subcontracting.portal.warehouse,stock.model_stock_warehouse,base.group_portal,1,0,0,0
+access_subcontracting_portal_procurement_group,subcontracting.portal.procurement.group,stock.model_procurement_group,base.group_portal,1,0,0,0
+access_subcontracting_portal_lot,subcontracting.portal.lot,stock.model_stock_lot,base.group_portal,1,0,1,0
+access_subcontracting_portal_putaway_rule,subcontracting.portal.putaway.rule,stock.model_stock_putaway_rule,base.group_portal,1,0,0,0
+access_subcontracting_portal_quant,subcontracting.portal.quant,stock.model_stock_quant,base.group_portal,1,0,0,0
+access_subcontracting_portal_location,subcontracting.portal.location,stock.model_stock_location,base.group_portal,1,0,0,0
+access_subcontracting_portal_production,subcontracting.portal.production,mrp.model_mrp_production,base.group_portal,1,1,1,0
```

Comment:
Maybe we should prevent the create because the subcontractor could create any `mrp.production` with himself as subcontractor. We could use a sudo in split_production for the serial backorder case